### PR TITLE
update docs for v15 release

### PR DESCRIPTION
**Pull request by jharenza**
#### What scientific question is your analysis addressing?
Update documentation for #543 

#### What was your approach?
update:
- download-data.sh
- release-notes.md
- data-formats.md
- data-files-descriptions.md

#### What GitHub issue does your pull request address?
#543 

#### Which areas should receive a particularly close look?
@cbethell - please take a look at the descriptions of the focal CN files in `data-formats.md` and `data-files-description.md` to be sure I have an accurate read of those.
@jaclyn-taroni let me know if the description of the GDC BED is OK.

#### Is there anything that you want to discuss further?
Once the documentation is acceptable, we can add the `data-files-description.md` and `release-notes.md` to our bucket, create a new `md5sum.txt` and test the download

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
Yes to documentation
No to download

### Data Release Checklist
- [x] Is the table in doc/data-file-descriptions.md up to date?
- [x] Is doc/data-format.md up to date?
- [x] Is doc/release-notes.md up to date?
- [x] Is download-data.sh up to date?

### CODE REVIEW

**Comment by jaclyn-taroni** [comment]
@jharenza is this in S3 yet?

**Comment by jharenza** [comment]
@jaclyn-taroni not yet - was awaiting document review. will update you in a bit!

File: doc/release-notes.md [hunk 1]
```diff
@@ -1,5 +1,101 @@
 # release notes
+# release notes
```
[the remark this answers is not in the document]
Comment (jharenza) [reply doc/release-notes.md]: fixed

**Comment by jharenza** [comment]
ok @jaclyn-taroni, I think we should be good to go. Downloads successfully and I included here the difference between md5sum files confirming the files changed.

NOTE to you and @cansavvy: I changed the name of `gencode.v19.basic.exome.hg38liftover.bed` to `gencode.v19.basic.exome.hg38liftover.100bp_padded.bed` to reflect that it was lifted then padded.

```
c85b76f88639:OpenPBTA-analysis harenzaj$ diff data/release-v14-20200203/md5sum.txt data/release-v15-20200228/md5sum.txt 
1d0
< ed528095ce371a64288fe4971cd77a30  data-files-description.md
13d11
< 1ccbc07e497e020d26676f5ad9f8fda2  pbta-cnv-consensus-gistic.zip
17,19d14
< d2f911ccc4bf80c2eed185fdacdc4283  pbta-fusion-putative-oncogenic.tsv
< b6027b28360f45b5064112594f810656  pbta-fusion-recurrently-fused-genes-byhistology.tsv
< aac74635d577f20fdc47e427cb119233  pbta-fusion-recurrently-fused-genes-bysample.tsv
48,50d42
< d855fd6550183204dd839d024be56e57  pbta-tcga-snv-lancet.vep.maf.gz
< 3474ccc3435bc633b986b673308de3dd  pbta-tcga-snv-mutect2.vep.maf.gz
< 39517b0c6d3dab29594aaf1aa2b04972  pbta-tcga-snv-strelka2.vep.maf.gz
60,61c52,64
< 6f68e51bd64c2a6a93fe5c0904cb7504  pbta-histologies.tsv
< 37ded33ca08fdfb25a1da0ae951f133f  release-notes.md
---
> 4feeec69576ad04414e3672e208646fd  consensus_seg_annotated_cn_autosomes.tsv.gz
> 40d8ccac9547f9a1964c2d802673edef  consensus_seg_annotated_cn_x_and_y.tsv.gz
> c9179107e413036ca02ed9b1090f52b2  data-files-description.md
> 1eac628848cf677a0b4234edb4885df0  gencode.v19.basic.exome.hg38liftover.100bp_padded.bed
> 1e9e3ec97d7f5544dbd8870afa07e73e  pbta-cnv-consensus-gistic.zip
> e5e1b089e7dd037ab59ba0bedc24a95f  pbta-fusion-putative-oncogenic.tsv
> 08543f50fa81ff51b19dd39d0dcb68dd  pbta-fusion-recurrently-fused-genes-byhistology.tsv
> 085076afcbfc0782c168effe3a5d8321  pbta-fusion-recurrently-fused-genes-bysample.tsv
> 626432b1b6be4ee3ac2ece9c29abde7c  pbta-histologies.tsv
> 1b38ca05adace463cbbcbf5faf0155b8  pbta-tcga-snv-lancet.vep.maf.gz
> 3474ccc3435bc633b986b673308de3dd  pbta-tcga-snv-mutect2.vep.maf.gz
> 39517b0c6d3dab29594aaf1aa2b04972  pbta-tcga-snv-strelka2.vep.maf.gz
> 75f61a765ece26ffa7fc1e281c3861df  release-notes.md
```

**Comment by jaclyn-taroni** [comment]
Ok, will get cracking on the CI stuff.

**Comment by jashapiro** [comment]
Changes made as part of #574 are done, and this passes CI. #578 is still open, but is not blocking this release.

**Pull request merged**